### PR TITLE
Add FakeIntakeParams to be able to disable LB from datadog-agent tests

### PR DESCRIPTION
**Pull request by KevinFairise2**
What does this PR do?
---------------------

Expose option to be able to disable Load Balancer from tests

Which scenarios this will impact?
-------------------

Motivation
----------

Additional Notes
----------------

### CODE REVIEW

File: scenarios/aws/fakeintake.go [hunk 1]
```diff
@@ -4,10 +4,11 @@ import (
 	"github.com/DataDog/test-infra-definitions/components/datadog/apps/fakeintake"
 	ddfakeintake "github.com/DataDog/test-infra-definitions/components/datadog/fakeintake"
 	resourcesAws "github.com/DataDog/test-infra-definitions/resources/aws"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/fakeintake/fakeintakeparams"
 )
 
-func NewEcsFakeintake(env resourcesAws.Environment) (*ddfakeintake.ConnectionExporter, error) {
-	fargateInstance, err := fakeintake.NewECSFargateInstance(env)
+func NewEcsFakeintake(env resourcesAws.Environment, options ...fakeintakeparams.Option) (*ddfakeintake.ConnectionExporter, error) {
```
Comment (pducolin) [inline scenarios/aws/fakeintake.go:10]: **👏 praise**
Exactly what I was thinking ! @pgimalac Can you rebase on this change and add `WithName` option ?

**Review by pducolin — APPROVED** [review]

File: scenarios/aws/fakeintake.go (hunk 1 above)
Comment (pgimalac) [reply scenarios/aws/fakeintake.go:10]: Haha I initially did it that way but ended up reverting to something simpler
But that's perfect, I'll rebase ! :)

**Comment by KevinFairise2** [comment]
> **💬 suggestion** Can we change the integration test to create a VM without LB ?

I can for sure

**Comment by KevinFairise2** [comment]
> **💬 suggestion** Can we change the integration test to create a VM without LB ?

After checking, the invoke task is configured to not use the load balancer by default. So the integration tests are actually not creating a load balancer with the VM

**Pull request merged**